bin/generate_schema.cmd

```diff
@@ -9,22 +9,20 @@ if "%VIRTUAL_ENV%"=="" (
 
 echo Generating Swagger schema
 python src\manage.py generate_swagger^
-    ./src/swagger2.0.json^
+    --output-file ./src/openapi.yaml^
     --overwrite^
-    --format=json^
+    --format=openapi^
     --mock-request^
     --url https://example.com/api/v1
 
-echo Converting Swagger to OpenAPI 3.0...
-call npm run convert
 call patch_content_types
 
 echo Generating unresolved OpenAPI 3.0 schema
 call use_external_components
 
```

Comment:
Zou je hier nog een stap willen toevoegen waarbij je [Prettier](https://prettier.io/) aanroept over het gegenereerde schema (o.a. met de `--single-quote` optie)? Prettier zou dan als npm requirement toegevoegd moeten worden aan de `package.json` van de betreffende API (net zoals `swagger2openapi`). `swagger2openapi` kan lijkt mij nu ook weg als dependency (ook in de betreffende API)?

Reply:
Ik heb prettier bij de dependencies gezet van package json.

En dan in de catalogi-api:

        "prettify": "npx prettier --write --single-quote ./src/openapi.yaml",

Reply:
"scripts": {
        "prettify": "npx prettier --write --single-quote ./src/openapi.yaml",
        "build": "npm run bundle",
        "bundle": "webpack",
        "watch": "webpack -w"
    },